docs/get-started/create-a-user-portal-account.md

```diff
@@ -115,3 +115,5 @@ under "NERC PI Account Request" option on **Help Topic** as shown in the image b
     help@nerc.mghpcc.org. You are going to get into NERC's ColdFront resource
     allocation management portal using the PI user role. If you want to request
```

Comment:
What do we mean "If you want to request a ticket"? Based on the information they have why would they request a ticket? The user does not know what a ticket would help them with. Are we trying to say something like "If you have any questions please create a ticket [here](https://mghpcc.supportsystem.com/open.php)"

Reply:
@joachimweyl : This is to inform user can also open a ticket to register as a PI user account. As NERC ticketing system has its own Help Topic called "**NERC Principal Investigator (PI) Account Request**" [here](https://mghpcc.supportsystem.com/open.php).

Reply:
The wording was slightly confusing hence why I suggested the other wording.

Reply:
@joachimweyl : This line is redundant and we already have information on how to request a PI user account by submitting a ticket on our ticketing system. I am removing this line.